test/Confluent.Kafka.IntegrationTests/Tests/Producer_Poll.cs

```diff
@@ -47,7 +47,8 @@ public void Producer_Poll(string bootstrapServers)
                 // call to Poll (or if the poll method hasn't been called, over
                 // the lifetime of the producer).
                 Assert.True(producer.Poll(TimeSpan.FromMilliseconds(500)) >= 1);
-                Assert.True(sw.ElapsedMilliseconds < 1);
+                var elapsed = sw.ElapsedMilliseconds;
+                Assert.True(elapsed < 2);
```

Comment:
Why is this asserting to less than 2 milliseconds when the comment says 500?

Reply:
the timeout on poll is 500 milliseconds (the operation should be close to instantaneous). previously, this test has never failed. now it is failing reliably. either the precision of Stopwatch has changed somehow or librdkafka is taking longer in poll.

Reply:
Measure (and debug=protocol) and ye shall find.

Also, 2ms does not yield a very robust test.

Reply:
that doesn't tell me.

agree the 2ms is not robust, but gonna leave it for now. Also, this is now passing with 1ms, underscoring your point.